sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/util/Beta.java

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
```

Comment:
Nice - thanks for taking care of this :-)

Reply:
np :-)